src/util/time.cpp

```diff
@@ -80,11 +70,12 @@ template <typename T>
 T GetTime()
 {
     const std::chrono::seconds mocktime{nMockTime.load(std::memory_order_relaxed)};
-
-    return std::chrono::duration_cast<T>(
+    const auto ret{
```

Comment:
`const T ret` ?

Dropping the `duration_cast` around `mocktime` means you can't use `GetTime<T>()` with `T` set to minutes or hours or similar. Seems fine, just noting.

Reply:
I think `auto` is fine for now. This will also reduce the diff in commit 8299fd2e9703a660ca948a5f29e9d3d2bf9abdf2, where the type is changed from `T` to `Clock::time_point::duration`.